workspaces/global-floating-action-button/plugins/global-floating-action-button/src/components/FAB.tsx

```diff
@@ -122,39 +124,39 @@ export const FAB = ({
         slotOptions[actionButton.slot || Slot.PAGE_END].tooltipDirection
       }
     >
-      <div className={className}>
-        <Fab
-          {...(newWindow ? { target: '_blank', rel: 'noopener' } : {})}
-          style={{ color: '#1f1f1f' }}
-          variant={
-            actionButton.showLabel || isExternal || !actionButton.icon
-              ? 'extended'
-              : 'circular'
+      <Fab
+        {...(newWindow ? { target: '_blank', rel: 'noopener' } : {})}
+        className={className}
+        style={{
+          color: iconColor ? iconColor : '#1f1f1f',
+          backgroundColor: !actionButton.color && !className ? 'white' : '',
```

Comment:
Can you undo this change and test ?
I tried it myself and it looks like this change is not needed